recipe/run_test.py

```diff
@@ -46,11 +46,6 @@
     print("Building on Windows?      ", WIN)
     print("Building on Linux?        ", LINUX)
     print("Building for PyPy?        ", PYPY)
-
-    if MIGRATING:
-        print("This is a migration, skipping test suite! Put it back later!", flush=True)
-        sys.exit(0)
-    else:
-        print("Running pytest with args")
-        print(PYTEST_ARGS, flush=True)
-        sys.exit(subprocess.call(PYTEST_ARGS, cwd=str(Path(IPython.__file__).parent)))
\ No newline at end of file
+    print("Running pytest with args")
```

Comment:
It'd be less error-prone to update run_test.py next time if you kept it the same as conda-forge

Reply:
you could do `MIGRATING=false run_test.py`, or `export MIGRATING=false` in the test commands

Reply:
`MIGRATING=false run_test.py` does not work

Reply:
I opted to remove everything since it still builds

Reply:
Added your second suggestion but am unsure why it won't work for windows